addons/l10n_si/tests/test_get_reference.py

```diff
@@ -0,0 +1,27 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class L10n_siInvoiceGetReferenceTest(AccountTestInvoicingCommon):
+
+    @classmethod
+    @AccountTestInvoicingCommon.setup_country('si')
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.invoice = cls.init_invoice('out_invoice', products=cls.product_a + cls.product_b)
+
+    def test_get_reference_slovenian_invoice(self):
+        self.assertFalse(self.invoice.payment_reference)
+        self.invoice.journal_id.invoice_reference_model = 'si'
+        self.invoice.action_post()
+        self.assertTrue(self.invoice.payment_reference)
```

Comment:
This is just testing that we have any `payment_reference`. It could literally be any string and it will work. We can do better than that :slightly_smiling_face:

Reply:
Well, we don't need this line anymore ...